src/mmw/mmw/urls.py

```diff
@@ -20,13 +19,9 @@
 
 admin.autodiscover()
 
-router = routers.DefaultRouter()
-router.register(r'users', apps.home.views.UserViewSet)
-
 urlpatterns = patterns(
     '',
     url(r'^', include(apps.home.urls)),
-    url(r'^api/', include(router.urls)),
```

Comment:
Interesting how our usage of DRF evolved. Instead of using the Viewset API we now just use the DRF decorators/serializers with generic Django view functions. Maybe we'll reap the benefits of the Viewset API on the next project...